tests/contrib/pytest/test_pytest.py

```diff
@@ -122,19 +144,19 @@ def __init__(self, name, value):
             def item_param():
                 return 42
 
+            circular_reference = A("circular_reference", A("child", None))
+            circular_reference.value.value = circular_reference
```

Comment:
```suggestion
            circular_reference.value = circular_reference
```

i think

Reply:
Does this make more sense?
```python
parent = A("parent", None)
child = A("child", parent)
parent.value = child
```

Reply:
`circular_reference.value` is the child object, and the child object's value is the reference back to the parent, let me know if I should make that more clear!

Reply:
ah yes! i didn't look closely enough at the `A` class